src/components/AudioPlayer/styles.scss

```diff
@@ -17,7 +17,7 @@
   }
 
   &.contacto-button--table-action-link {
-    background-color: var(--background-highlight-color);
+    background-color: var(--background-highlight-color) !important;
```

Comment:
is there no other way to apply this style instead of !important

Reply:
The style was already there but it was getting over-written so added this